tests/_support/Page/Acceptance/Administrator/ControlPanelPage.php

```diff
@@ -0,0 +1,13 @@
+<?php
+namespace Page\Acceptance\Administrator;
+
+use Page\Acceptance\Administrator\AdminPage;
+
+class ControlPanelPage extends AdminPage
+{
+    public static $url = "/administrator/index.php";
+
+    public static $pageTitle = 'Control Panel';
+
+    public static $pageTitleContext = ['class' => 'page-title'];
```

Comment:
I am probably still for an array here.. As else wise we are going to have thousands of public static (readability?) in pages.

```php
public static $sel = [
   'url'  => '/administrator/index.php',
   'pageTitle' => 'Control Panel',
   'pageTitleContext' => ['class' => 'page-title'];
];
```

(Better indention on the elements, GitHub does not allow me to)

// cc @javigomez, @pritalpatel, @nibra, @puneet0191